#coding=utf-8

__all__ = ['ttypes', 'constants', 'IActivityAdminServiceGetAdminActivityList', 'IActivityAdminServiceAddAdminActivity', 'IActivityAdminServiceEditAdminActivity', 'IActivityAdminServiceGetAdminActivityById', 'IActivityAdminServiceUpdateAdminActivityStatus', 'IActivityAdminServiceDeleteAdminActivity', 'IActivityAdminServiceGetAdminPostListByActivityId', 'IActivityAdminServiceAddAdminPrize', 'IActivityAdminServiceDeleteAdminPrize', 'IActivityAdminServiceUpdateAdminPrizes', 'IActivityAdminServiceGetAdminPrizeList', 'IActivityAdminServiceConfirmPrizeList']
